### ISSUE TITLE
Landing page

### ISSUE THREAD

**Issue by salexzee** [TribalDev/tribal_blog#9]
The landing page needs to be designed and styled.

**Comment by salexzee** [comment]
We need something under the landing hero image that is interesting and gives people a reason to scroll down there. Any ideas?

**Comment by jaeming** [comment]
Yes. aside from the paragraph I've written there we could always do the three prong mission thing that nearly everyone does. Or we could just put up a grid of categories from our posts...
